buildconfig/Jenkins/Conda/pixi-utils

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Download and install pixi (if it isn't installed)
+install_pixi() {
+  # check if pixi is not installed
+  if ! command -v pixi >/dev/null 2>&1
+  then
+    curl -fsSL https://pixi.sh/install.sh | sh
+    # Verify pixi was installed successfully
+    if ! command -v $HOME/.pixi/bin/pixi >/dev/null 2>&1; then
+      echo "ERROR: pixi installation failed"
+      exit 1
+    fi
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+      export PATH=$PATH:$HOME/.pixi/bin
+    else
+      if [ -f ~/.bashrc ]; then
+        source ~/.bashrc
+      else
+        export PATH=$PATH:$HOME/.pixi/bin
+      fi
+    fi
```

Comment:
```suggestion
    if [[ "$OSTYPE" != darwin* && -f ~/.bashrc ]]; then
      source ~/.bashrc
    else
      export PATH="$PATH:$HOME/.pixi/bin"
    fi
```
Can be refactored to avoid duplication.